src/eko/evolution_operator/flavors.py

```diff
@@ -130,7 +131,7 @@ def rotate_pm_to_flavor(label):
     return l
 
 
-def rotate_matching(nf, inverse=False):
+def rotate_matching(nf, is_qed=False, inverse=False):
```

Comment:
I would also go for the short version, just `qed`. It is a flag, so it should be non-ambiguous :)